app/src/main/java/com/randomappsinc/studentpicker/home/NameListsAdapter.java

```diff
@@ -117,19 +126,24 @@ public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which)
                         preferencesManager.removeNameList(getItem(position));
                         content.remove(position);
                         setNoContent();
-                        notifyDataSetChanged();
+                        notifyItemRemoved(position);
+                        notifyItemRangeChanged(position, content.size());
```

Comment:
I don't think we need the notifyItemRangeChanged. We're only removing 1 item right? We aren't updating several.

Reply:
when I used the notifyItemRemoved the items still in their place so I have to rearrange them.

Reply:
Interesting... I've written this code before and I don't remember having to do that. I wonder if it's an Android 10 issue or something wrong with my old, janky code lol.